mozilla_schema_generator/configs/glean.yaml

```diff
@@ -5,179 +5,153 @@ metrics:
         not:
           - glean_ping_info
       type: boolean
-      labeled: false
```

Comment:
I have no idea what these bits are about but it looks like they may be the reason CI is failing.

Reply:
Right, it is. PR passes so we should be good to go.